R/pk-parameter.R

```diff
@@ -0,0 +1,24 @@
+
+#' @title PKParameter
+#' @docType class
+#' @description  Results of a simulation run (either individual or population simulation)
+PKParameter <- R6Class("PKParameter",

```

Comment:
A PKParameter in Core has
* a QuantityPath (path of underlying quantity, without simulation name). So,methng like "Organim|Liver|Cell|drug"
* a name (parmaeter name such as AUC, Tmax etc..)
* values. An array of double. One entry for each individual associated with the simulation (for individual simulation, R will return a single value)

Reply:
Please use R6::R6Class for consistency.